docker-compose.yml

```diff
@@ -188,6 +200,7 @@ services:
       --defaultentrypoints=http,https \
       --entrypoints='Name:http Address::80' \
       --entrypoints='Name:https Address::443 TLS:/ssl/test.crt,/ssl/test.key' \
+      --entrypoints='Name:service Address::8080' \
```

Comment:
why do we need an extra entrypoint?

Reply:
because of token issuer problems. Note that 8080 is the standard port.